### PR TITLE
Adds plugin for stage 3 class properties.

### PR DESCRIPTION
**Pull request by weerd**
This PR adds a plugin to allow for [class properties](https://github.com/babel/babel/tree/master/packages/babel-plugin-proposal-class-properties#example) which are sweet and help simplify our React JS code.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,11 +13,12 @@
   "author": "David Furnes <dfurnes@dosomething.org>",
   "license": "MIT",
   "dependencies": {
-    "babel-preset-env": "^1.6.1",
+    "@babel/plugin-proposal-class-properties": "^7.0.0-beta.40",
```
Comment (DFurnes) [inline package.json]: Is it safe to install `7.x`  plugins with Babel 6? Might want to drop to `^6.0.0` to be safe.

Comment (weerd) [reply package.json]: Yeah, I was wondering about that too 😬 Okie dokie.

File: index.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ module.exports = {
     require('babel-plugin-lodash'),
     require('babel-plugin-transform-export-extensions'),
     require('babel-plugin-transform-object-rest-spread'),
+    require('@babel/plugin-proposal-class-properties'),
```
Comment (DFurnes) [inline index.js]: Change this to `babel-plugin-transform-class-properties`!

Comment (weerd) [reply index.js]: oh dang. Forgot there was another spot 😬

**Pull request merged**